version.php

```diff
@@ -30,3 +30,5 @@
 $plugin->component = 'report_apocalypse';
 $plugin->maturity  = MATURITY_STABLE;
 $plugin->release   = '1.3';
+$plugin->supported = [401]; 
+$plugin->incompatible = [402, 403]; // no longer supported after Moodle 4.1
```

Comment:
Please test this, I think this needs to be

$plugin->incompatible = 402

https://moodledev.io/docs/4.4/apis/commonfiles/version.php#incompatible-versions

Reply:
Thanks for sharing. I've updated and tested it. It passes the plugin validation check but fails in the plugin check, which will prevent the plugin from being installed.

![Screenshot from 2024-07-18 16-12-02](https://github.com/user-attachments/assets/22dc584d-fd33-4918-b0ec-84c395dab82f)